packages/core/package.json

```diff
@@ -9,10 +9,9 @@
     "/build",
     "/dev",
     "/dist",
-    "/entries",
     "/fix",
     "/index",
-    "/mocks",
+    "/logger",
```

Comment:
I think these need to stay in. Entries and mocks are important to `build`/`start`, and should be part of the npm package.